examples/addr2line.rs

```diff
@@ -94,90 +95,60 @@ fn load_file_section<'input, 'arena, Endian: gimli::Endianity>(
     }
 }
 
+#[derive(Parser)]
+#[clap(bin_name = "hardliner", name = "hardliner")]
```

Comment:
The binary that rust builds is named `addr2line`, so this isn't right. Let's remove the `hardliner` name completely, since it's an old name that is no longer used anywhere.